# DP: Fix libjava testsuite with dejagnu 1.6, taken from the trunk.

2016-04-20  Ben Elliston  <bje@gnu.org>

	* testsuite/lib/libjava.exp (libjava_arguments): Use 'file
	normalize' rather than the defunct DejaGnu 'absolute' proc.

 
--- a/src/libjava/testsuite/lib/libjava.exp
+++ b/src/libjava/testsuite/lib/libjava.exp
@@ -416,7 +416,7 @@
 
     # Search for libtool.  We need it to link.
     set found_compiler 0
-    set d [absolute $objdir]
+    set d [file normalize $objdir]
     foreach x {. .. ../.. ../../..} {
 	if {[file exists $d/$x/libtool]} then {
 	    # We have to run silently to avoid DejaGNU lossage.
